src/main/rx/rx_spi.c

```diff
@@ -67,6 +67,12 @@ static protocolDataReceivedFnPtr protocolDataReceived;
 static protocolProcessFrameFnPtr protocolProcessFrame;
 static protocolSetRcDataFromPayloadFnPtr protocolSetRcDataFromPayload;
 
+#ifdef USE_RX_EXPRESSLRS
+typedef void (*protocolStopFnPtr)(void);
+static void nullProtocolStop(void) {}
+static protocolStopFnPtr protocolStop = nullProtocolStop;
+#endif
+
```

Comment:
It is a little bit tricky to add `protocolStopFnPtr` this way. How about adding it as another property like `protocolSetRcDataFromPayloadFnPtr` and all others? You can initialize it to NULL or nullProtocolStop in the begging of `rxSpiSetProtocol`. Maybe it will add few bytes but it is more consistent solution.

Reply:
Thanks for looking, I will fix it in a few hours